public/app/features/admin/ldap/LdapDrawer.tsx

```diff
@@ -0,0 +1,251 @@
+      description={t('ldap-drawer.group-mapping-section.description', 'Map LDAP groups to Grafana org roles')}
+    >
+      <Trans i18nKey="ldap-drawer.group-mapping-section.label">Group mapping</Trans>
+    </Label>
+  );
+
+  const useTlsDescription = (
+    <Trans i18nKey="ldap-drawer.extra-security-section.use-ssl.tooltip">
+      For a complete list of supported ciphers and TLS versions, refer to:{' '}
+      {
+        <TextLink style={{ fontSize: 'inherit' }} href="https://go.dev/src/crypto/tls/cipher_suites.go" external>
+          https://go.dev/src/crypto/tls/cipher_suites.go
+        </TextLink>
+      }
+    </Trans>
+  );
+
+  return (
+    <Drawer title={t('ldap-drawer.title', 'Advanced Settings')} onClose={onClose}>
+      <CollapsableSection label={t('ldap-drawer.misc-section.label', 'Misc')} isOpen={true}>
+        <Field
+          label={t('ldap-drawer.misc-section.allow-sign-up.label', 'Allow sign up')}
+          description={t(
+            'ldap-drawer.misc-section.allow-sign-up.descrition',
+            'If not enabled, only existing Grafana users can log in using LDAP'
+          )}
+        >
+          <Switch id="allow-sign-up" {...register('settings.allowSignUp')} />
+        </Field>
+        <Field
+          label={t('ldap-drawer.misc-section.port.label', 'Port')}
+          description={t(
+            'ldap-drawer.misc-section.port.description',
+            'Default port is 389 without SSL or 636 with SSL'
+          )}
+        >
+          <Input
+            id="port"
+            placeholder={t('ldap-drawer.misc-section.port.placeholder', '389')}
+            type="number"
+            {...register('settings.config.servers.0.port', { valueAsNumber: true })}
+          />
+        </Field>
+        <Field
+          label={t('ldap-drawer.misc-section.timeout.label', 'Timeout')}
+          description={t(
+            'ldap-drawer.misc-section.timeout.description',
+            'Timeout in seconds for the connection to the LDAP server'
+          )}
+        >
+          <Input
+            id="timeout"
+            placeholder={t('ldap-drawer.misc-section.timeout.placeholder', '389')}
+            type="number"
+            {...register('settings.config.servers.0.timeout', { valueAsNumber: true })}
+          />
+        </Field>
+      </CollapsableSection>
+      <CollapsableSection label={t('ldap-drawer.attributes-section.label', 'Attributes')} isOpen={true}>
+        <Text color="secondary">
+          <Trans i18nKey="ldap-drawer.attributes-section.description">
+            Specify the LDAP attributes that map to the user&lsquo;s given name, surname, and email address, ensuring
+            the application correctly retrieves and displays user information.
+          </Trans>
+        </Text>
+        <Field htmlFor="name" label={t('ldap-drawer.attributes-section.name.label', 'Name')}>
+          <Input id="name" {...register('settings.config.servers.0.attributes.name')} />
```

Comment:
react has a `useId()` hook that can help you generate ids: https://react.dev/reference/react/useId

problem here is that id is supposed to be unique on a page. but when the ids are so generic, like `name`, it's only a matter of time before there's more than one on a page and this linking might break